ansible/roles/debops.postfix_virtual_ldap/defaults/main.yml

```diff
@@ -0,0 +1,685 @@
+  - name: 'virtual_alias_maps'
+    value: '{{ postfix_virtual_ldap__virtual_alias_maps | join(",") }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  #A prefix that the virtual delivery agent prepends to all pathname results from $virtual_mailbox_maps table lookups.
+  # This is a safety measure to ensure that an out of control map doesn't litter the file system with mailboxes.
+  - name: 'virtual_mailbox_base'
+    value: '{{ postfix_virtual_ldap__mailbox_base }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Optional lookup tables with all valid addresses in the domains that match $virtual_mailbox_domains.
+  - name: 'virtual_mailbox_maps'
+    value: 'ldap:/etc/postfix/ldap_virtual_recipients.cf'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Lookup tables with the per-recipient user ID that the virtual delivery agent uses while writing
+  # to the recipient's mailbox.
+  - name: 'virtual_uid_maps'
+    value: 'static:{{ postfix_virtual_ldap__vmail_posix_uidnumber }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # Lookup tables with the per-recipient group ID for virtual mailbox delivery.
+  - name: 'virtual_gid_maps'
+    value: 'static:{{ postfix_virtual_ldap__vmail_posix_gidnumber }}'
+    section: 'virtual'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool and
+                postfix__version is version_compare("2.0", ">="))
+            else "absent" }}'
+
+  # The minimum user ID value that the virtual delivery agent accepts as a result from $virtual_uid_maps table lookup
+  - name: 'virtual_minimum_uid'
+    value: '{{ postfix_virtual_ldap__vmail_posix_uidnumber }}'
+    section: 'virtual'
+    state: 'absent'
+
+  # Optional lookup table with the SASL login names that own the sender (MAIL FROM) addresses.
+  - name: 'smtpd_sender_login_maps'
+    value: 'ldap:/etc/postfix/ldap_virtual_senders.cf'
+    section: 'base'
+    state: '{{ "present"
+            if (postfix_virtual_ldap__ldap_enabled|bool)
+            else "absent" }}'
+                                                                   # ]]]
+# .. envvar:: postfix_virtual_ldap__postfix__lookup_header [[[
+#
+# The header of the lookup table containing the settings to connect to the LDAP server
+postfix_virtual_ldap__postfix__lookup_header:
```

Comment:
renamed var, It was a copy-pasta typo

> Also, that's an interesting solution for having standardized header for all `ldap_*` lookup tables. Since `debops.postfix` role does not have a dedicated solution yet, it can stay for now. :-)

;-)  I do not like code which repeats and is error prone.
If there is an easier method, please give me hint :)